tests/test_simple_unit.py

```diff
@@ -11,15 +11,17 @@
 import pyparsing as pp
 from collections import namedtuple
 from datetime import datetime
+
 ppt = pp.pyparsing_test
+TestParseResultsAsserts = ppt.TestParseResultsAsserts
 
 # Test spec data class for specifying simple pyparsing test cases
 PpTestSpec = namedtuple("PpTestSpec", "desc expr text parse_fn "
                                       "expected_list expected_dict expected_fail_locn")
 PpTestSpec.__new__.__defaults__ = ('', pp.Empty(), '', 'parseString', None, None, None)
 
 
-class PyparsingExpressionTestCase(ppt.ParseResultsAsserts, unittest.TestCase):
+class PyparsingExpressionTestCase(TestParseResultsAsserts):
```

Comment:
since `TestParseResultsAsserts` extends `TestCase` we do not need the multiple inheritance (plus it was confusing pyCharm's simple test discovery logic).